### PR TITLE
Made additional changes to allow SSL connection.

### PR DESCRIPTION
**Pull request by ThomasOverbergh**
In relation to Issue #1 

Can now successfully connect to our Irods server. 
Already able to create folders. 
Some issues with the other commands (Write, Create,...), will look at this next.

### CODE REVIEW

File: iRods_Csharp/irods-Csharp/Account.cs
Comment (ThomasOverbergh) [inline iRods_Csharp/irods-Csharp/Account.cs:1]: Updated this to use rods 4.3.0

File: iRods_Csharp/irods-Csharp/Connection.cs
Comment (ThomasOverbergh) [inline iRods_Csharp/irods-Csharp/Connection.cs:1]: `_serverStream = secureServerStream` needed to be moved up.

File: iRods_Csharp/irods-Csharp/Structs/Structs.cs
Comment (ThomasOverbergh) [inline iRods_Csharp/irods-Csharp/Structs/Structs.cs:1]: Added two constructors without arguments. Not sure if still needed...

File: iRods_Csharp/irods-Csharp/Account.cs [hunk 1]
```diff
@@ -60,9 +60,11 @@ public Account(string host, int port, string home, string user_name, string zone
 
     /// <summary>
     /// Creates startup pack used to establish connection with server
+    /// Thomas Overbergh updated this to use rods 4.3.0.
     /// </summary>
     /// <returns>StartupPack_PI Irods Message</returns>
-    public StartupPackPi MakeStartupPack(string option = "") => new (Options.iRODSProt_t.XML_PROT, 0, 0, _userName, _zoneName, _userName, _zoneName, "rods4.2.6", "d", option);
+    public StartupPackPi MakeStartupPack(string option = "") => new(Options.iRODSProt_t.XML_PROT, 0, 0, _userName, _zoneName, _userName, _zoneName, "rods4.3.0", "d", option);
+    //public StartupPackPi MakeStartupPack(string option = "") => new (Options.iRODSProt_t.XML_PROT, 0, 0, _userName, _zoneName, _userName, _zoneName, "rods4.2.6", "d", option);
```
Comment (JTeeuwissen) [inline iRods_Csharp/irods-Csharp/Account.cs]:
```suggestion
```

File: iRods_Csharp/irods-Csharp/Structs/Structs.cs
Comment (JTeeuwissen) [reply iRods_Csharp/irods-Csharp/Structs/Structs.cs:1]: You can leave them, they don't hurt anybody.

File: iRods_Csharp/irods-Csharp/Structs/Structs.cs [hunk 2]
```diff
@@ -218,7 +223,8 @@ public CsNegPi()
     public CsNegPi(ClientServerPolicyResult policyResult)
     {
         Status = policyResult is ClientServerPolicyResult.Failure ? 0 : 1;
-        Result = $"CS_NEG_RESULT_KW={ClientServerPolicyToString(policyResult)}";
+       // Result = $"CS_NEG_RESULT_KW={ClientServerPolicyToString(policyResult)}"; //NOK
+        Result = $"cs_neg_result_kw={ClientServerPolicyToString(policyResult)};"; //OK
```
Comment (JTeeuwissen) [inline iRods_Csharp/irods-Csharp/Structs/Structs.cs]:
```suggestion
        Result = $"cs_neg_result_kw={ClientServerPolicyToString(policyResult)};";
```
🤦 silly me.

File: iRods_Csharp/irods-Csharp/Connection.cs
Comment (JTeeuwissen) [reply iRods_Csharp/irods-Csharp/Connection.cs:1]: Makes sense

**Review by JTeeuwissen — CHANGES_REQUESTED** [review]

**Comment by JTeeuwissen** [comment]
So you're telling me the only problem with TLS and stuff was the message casing? 😮‍💨 .

**Comment by ThomasOverbergh** [comment]
> So you're telling me the only problem with TLS and stuff was the message casing? 😮‍💨 .

I'm afraid so ;-)

**Pull request merged**